src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -227,7 +227,10 @@ class StockMovement {
                 shipment: shipment,
                 isFromOrder: shipment?.isFromPurchaseOrder,
                 isShipped: shipment?.status?.code >= ShipmentStatusCode.SHIPPED,
-                isReceived: shipment?.status?.code >= ShipmentStatusCode.RECEIVED
+                isReceived: shipment?.status?.code >= ShipmentStatusCode.RECEIVED,
+                driverName: shipment.driverName,
+                trackingNumber: shipment.referenceNumbers.first(),
```

Comment:
We should be looking this up the same way we look it up when we're going to edit the tracking number. Also, shouldn't the tracking number be a string, not an object?

Reply:
@jmiranda I have updated PR